airbyte_cdk/sources/declarative/retrievers/file_uploader/base_file_uploader.py

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+#
+
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+
+from airbyte_cdk.sources.declarative.types import Record
+
+
+@dataclass
+class BaseFileUploader(ABC):
```

Comment:
`Base` does not seem in our vocabulary for interfaces. Could we have something like this? 

* Parent: FileUploader
* DefaultFileUploader (this naming is not very good but we have history of naming things as `Default` when we don't know (like [here](https://github.com/airbytehq/airbyte-python-cdk/blob/2f29eff3f5d74862e1667f6274a0f23d1b3a6c0c/airbyte_cdk/sources/declarative/requesters/paginators/default_paginator.py#L33))
* ConnectorBuilderFileUploader